package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Should this have content? Do we need it? (same for outermost lock file)

Reply:
This file shouldn't exist, my bad, thanks @janniks